keyboards/whtkbrd/readme.md

```diff
@@ -0,0 +1,15 @@
+# Whtkbrd
+
+![Whtkbrd](https://i.imgur.com/xQbjgjS.jpeg)
+
+Split keyboard for Kailh Choc switches based on STM32F072. Heavily inspired by [Keyseebee](https://github.com/TeXitoi/keyseebee).
+
+* Keyboard Maintainer: [Alexander Metz](https://github.com/wahtak)
+* Hardware Supported: [Whtkbrd](https://oshwlab.com/whtk/keeb1)
+* Hardware Availability: Fabrication with JLCPCB
+
+Make example for this keyboard (after setting up your build environment):
```

Comment:
Could you add instructions on how to enter the bootloader, to flash new firmware?  This is needed, as a new user may not be aware of how to enter to bootloader to flash new firmware.

Eg: 
```markup
## Bootloader

Enter the bootloader in 3 ways:

* **Bootmagic reset**: Hold down the key at (0,0) in the matrix (usually the top left key or Escape) and plug in the keyboard
* **Physical reset button**: Briefly press the button on the back of the PCB - some may have pads you must short instead
* **Keycode in layout**: Press the key mapped to `RESET` if it is available
```

Reply:
Thanks for the suggestion, I extended the readme with bootloader and flashing instructions.